src/gt4py/backend/dawn_backends.py

```diff
@@ -107,19 +138,38 @@ def visit_VarRef(self, node: gt_ir.VarRef, **kwargs):
         return sir_utils.make_var_access_expr(name=node.name, is_external=True)
 
     def visit_FieldRef(self, node: gt_ir.FieldRef, **kwargs):
-        offset = [node.offset[ax] for ax in DOMAIN_AXES]
+        offset = [node.offset[ax] if ax in node.offset else 0 for ax in DOMAIN_AXES]
         return sir_utils.make_field_access_expr(name=node.name, offset=offset)
 
     def visit_UnaryOpExpr(self, node: gt_ir.UnaryOpExpr, **kwargs):
-        op = node.op.python_symbol
+        op = self.visit_Operator(node.op)
         operand = self.visit(node.arg)
         return sir_utils.make_unary_operator(op, operand)
 
     def visit_BinOpExpr(self, node: gt_ir.BinOpExpr, **kwargs):
         left = self.visit(node.lhs)
         right = self.visit(node.rhs)
-        op = node.op.python_symbol
-        return sir_utils.make_binary_operator(left, op, right)
+        if node.op.python_symbol == "**":
+            sir = self.visit_ExpOpExpr(left, right)
+        else:
+            op = self.visit_Operator(node.op)
+            sir = sir_utils.make_binary_operator(left, op, right)
+        return sir
+
+    def visit_ExpOpExpr(self, left, right):
```

Comment:
This is a helper method not really a visit method for `ExpOpExpr` nodes, so it should be renamed to make this explicit (like `make_pow_expr` or something similar)

Reply:
Changed to a helper method.